.styleci.yml

```diff
@@ -0,0 +1,18 @@
+preset: symfony
+
+enabled:
+  - align_double_arrow
+  - newline_after_open_tag
+  - ordered_use
+  - long_array_syntax
+  - php_unit_construct
+  - php_unit_strict
+
+disabled:
+  - psr0
```

Comment:
The fact that StyleCI is exclusively using php-cs-fixer atm, is just a happy coincidence I guess. We're going to add our own custom tools into the mix to give even better results where php-cs-fixer can't do stuff, say, like fixing indentation well.